### ISSUE TITLE
training problem 

### ISSUE THREAD

**Issue by chuanruihu** [likelyzhao/peron_attribute#2]
when I training follow your step,, in final the loss does not decrease~  Thank you for your reply.

**Comment by likelyzhao** [comment]
there are something wrong with training code ,pls pull the latest training code

**Comment by chuanruihu** [comment]
Thank you for your reply  , when i use your again , the same problem is shown again. I hope you to show detail how to use your code.Thank you
